pkg/apiserver/watch.go

```diff
@@ -181,11 +182,20 @@ func (s *WatchServer) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	w.WriteHeader(http.StatusOK)
 	flusher.Flush()
 
+	delay := wait.NewChannelDelayedAction(
+		wait.Interval{Count: 0, Interval: 5 * time.Millisecond},
+		wait.Interval{Count: 10, Interval: 100 * time.Millisecond},
+		wait.Interval{Count: 100, Interval: 500 * time.Millisecond},
```

Comment:
I'm wondering where those values come from. Where do they come from?

My personal feeling is that 500ms may be too high, but if those are based on some experiment, can you please describe it?